### ISSUE TITLE
[ISSUE]    Adding ArcoLinux, ubuntu, Debian linux hosts  BASH shell is not discovered  only SH and DASH 

### ISSUE THREAD

**Issue by dwilliam62** [xpipe-io/xpipe#126]
During the host scan after 1st logging in with Xpipe  and all options are selected  the BASH shell it not discovered. 
Ony SH and DASH    I have to edit it to change to /bin/bash 

This happens with ARCH,  Debian, ubuntu but works correctly with a Synology NAS device 

In one host it shows BASH and the version but /bin/sh was still the selected shell to execute 

![image](https://github.com/xpipe-io/xpipe/assets/129223418/a1540f07-ef61-464d-a084-b4f3940a05e8)

**Comment by crschnick** [comment]
Yes it seems like there has been a misunderstanding with the interface that many people had. It does not explicitly add your log in shell because you can already open it if you click on the ssh connection itself, that does put you into a bash shell. The original goal was to avoid duplication, but I just realized that this makes things more complicated to understand.

I will just add the login shell as well to the shell environments in the next update, that should make things clearer.

**Comment by dwilliam62** [comment]
Thank you.  In my case, maybe I was doing it wrong but I was getting just
/bin/sh when I connected.   So all my functions and aliases
obviously weren't there.

**Comment by crschnick** [comment]
If it did not put you into the right login shell, then that is a bug. So to confirm, if you click on the SSH connection itself, not the shell environments, it puts you into /bin/sh?

**Comment by dwilliam62** [comment]
Let me try again. I might have NOT been starting it correctly.    PEBKAC

Thank you for this

**Comment by dwilliam62** [comment]
My apologies.  100% PEBKAC.    I looked at the SSH icon as a classification not a launcher.     thank you

**Comment by crschnick** [comment]
To make this more clear, it will also now discover the default shell.
